python/dalex/NEWS.md

```diff
@@ -1,5 +1,8 @@
 dalex (development)
 ----------------------------------------------------------------
+* *feature:* parameter `N` in `model_profile` can be set to `None`, to select all observations 
+* *fix:* `label` and `groups` fixed in `model_profile`
```

Comment:
It would be best to write what exactly was fixed if there is no issue explaining this problem.

Reply:
done

Reply:
So can we delete line 4? It is not informative at all.